.s2i/bin/run

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+
+set -e
+
+if [[ "$DATABASE_HOST" =~ '_SERVICE_HOST' ]]; then
+    DATABASE_HOST=$(echo "$DATABASE_HOST" | tr '[:lower:]' '[:upper:]')
+    export DATABASE_HOST=${!DATABASE_HOST}
+    echo "---> DATABASE_HOST: ${DATABASE_HOST} ..."
+fi
+
+if [[ "$DATABASE_PORT" =~ '_SERVICE_PORT' ]]; then
+    DATABASE_PORT=$(echo "$DATABASE_PORT" | tr '[:lower:]' '[:upper:]')
+    export DATABASE_PORT=${!DATABASE_PORT}
+    echo "---> DATABASE_PORT: ${DATABASE_PORT} ..."
+fi
+
+if [ -z "${DO_NOT_CREATE_DATABASE}" ]; then
+    echo "---> Creating database ..."
+    bundle exec rake db:create
+fi
+
+if [ -z "${DO_NOT_MIGRATE}" ]; then
+    echo "---> Migrating database ..."
+    bundle exec rake db:migrate
+fi
+
+if [ -z "${DO_NOT_SEED}" ]; then
+    echo "---> Seeding database ..."
+    set +e
+    bundle exec rake db:seed
+    set -e
+fi
+
+export RACK_ENV=${RACK_ENV:-"production"}
+
+exec bundle exec "unicorn -c ./deployment/heroku/unicorn.rb --listen 0.0.0.0:8080"
```

Comment:
Can we move executed command to an optional environment variable? The Heroku unicorn configuration runs the background jobs in a thread within the unicorn server which is fine for getting started but does not allow scaling. As I understand how Openshift works one could then set `WORKER_CMD` to `unicorn --listen 0.0.0.0:8080` and `bin/threaded.rb` in two different pods.

Reply:
@dsander Good idea -- it's done.